docs/source/guide/webhooks.md

```diff
@@ -0,0 +1,143 @@
+    <td>Use to prompt model retraining. </td>
+  </tr>
+  <tr>
+    <td>[Annotation Deleted](webhook_reference.html#Annotation-Deleted)</td>
+    <td>For a specific project, triggers when an annotation is deleted.</td>
+    <td>Use to create a new version of a training dataset. </td>
+  </tr>
+  <tr>
+    <td>[Project Created](webhook_reference.html#Project-Created)</td>
+    <td>For an organization, triggers when a project is created.</td>
+    <td>Use to create a new pipeline for data management.</td>
+  </tr>
+  <tr>
+    <td>[Project Updated](webhook_reference.html#Project-Updated)</td>
+    <td>For an organization, triggers when project settings are updated, created, or saved.</td>
+    <td>Use to update an existing data management pipeline.</td>
+  </tr>
+  <tr>
+    <td>[Project Deleted](webhook_reference.html#Project-Deleted)</td>
+    <td>For an organization, triggers when a project is deleted.</td>
+    <td>Use to remove a data management pipeline. </td>
+  </tr>
+</table>
+
+## How to integrate with webhooks
+
+To integrate with webhooks in Label Studio, you must do the following:
+1. Create a webhook endpoint or URL as part of your application or machine learning model pipeline.
+2. Make sure the webhook endpoint or URL is accessible to your Label Studio instance and can accept HTTP POST requests.
+3. [Add the webhook to Label Studio](webhooks.html#Add-a-new-webhook-in-Label-Studio).
+
+Set up your webhook endpoint to read the event payload from Label Studio and take action based on the event.
+
+## Add a new webhook in Label Studio
+
+Set up a webhook URL in Label Studio and associate it with one or more event triggers. You can set up a webhook URL in the Label Studio UI, or using the API. You can configure as many webhook connections as you want, but too many might overwhelm your instance. 
+
+### Add a webhook in Label Studio UI
+
+Add a webhook URL to Label Studio. The webhook URL must be set up to accept HTTP POST requests.
+
+1. In the Label Studio UI, open the project that you want to associate with a webhook URL.
+2. Click **Settings** and click **Webhooks**.
+3. Click **New Webhook**. 
+4. In the **Payload URL** field, provide the URL to send event payloads to. For example, `https://www.example.com/webhook`.
+5. (Optional) Toggle the **Is Active** option to deactivate the webhook until it is ready to use. Otherwise, the webhook becomes active as soon as you save it and events are sent to the URL. 
+6. (Optional) Add any headers required by the webhook URL. Specify the header name and the value. You can use headers to authenticate a request to your webhook URL. You can't add more than 10 headers. For example, `Authorization` and `Basic bGFiZWxzdHVkaW86ZXhhbXBsZQ==`
+7. Select whether to send a payload with the event. For example, choose to send the payload for `Annotation created` events to update the correct pipeline based on the project ID sent in the payload.  
```

Comment:
Could 7-8 be optional too? Also for optional steps, defaults should be clarified

Reply:
done thanks